pkg/kvstore/etcd.go

```diff
@@ -223,7 +223,10 @@ func newEtcdClient(config *client.Config, cfgPath string) (BackendOperations, er
 	if err != nil {
 		return nil, err
 	}
-	log.Info("Waiting for etcd client to be ready")
+	log.WithFields(logrus.Fields{
+		"endpoints": config.Endpoints,
```

Comment:
config can be nil - this would panic if it is nil.

Reply:
The branch in that function is actually dead code. The only caller of `newEtcdClient()` does:
```
          if e.config == nil {
                  if !endpointsSet && !configSet {
                          return nil, fmt.Errorf("invalid etcd configuration, %s or %s must be specified",
                                  EtcdOptionConfig, addrOption)
                  }

                  e.config = &client.Config{}
```

Reply:
I will remove the dead code